app/serializers/basic_category_serializer.rb

```diff
@@ -19,7 +19,7 @@ class BasicCategorySerializer < ApplicationSerializer
              :notification_level,
              :can_edit,
              :topic_template,
-             :has_children,
+             :has_children?,
```

Comment:
Is `?` allowed in the attribute name? Im wondering if there is an existing test for this.

Reply:
There was a test, but it was broken. The serializer types were "string/null", but it should've been "bool".

Attribute names containing characters such as a question mark are sanitized and only `has_children` is returned. The question mark is necessary because we want to serialize `category.has_children?` (which exists) instead of `category.has_children` (which does not exist).